### PR TITLE
fix annotations for types generation

### PR DESCRIPTION
**Pull request by tbo47**
Follow up  of https://github.com/openglobus/openglobus/pull/384

### CODE REVIEW

File: src/og/Extent.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ export class Extent {
     /**
      * Creates bound extent instance by coordinate array.
      * @static
-     * @param {Array.<og.LonLat>} arr - Coordinate array.
```
Comment (yeluoyunfei) [inline src/og/Extent.js]: it will has issue for array in type script, see
image

Comment (tbo47) [reply src/og/Extent.js]: Yes. It is what this PR is fixing. Am I missing something here?

Comment (tbo47) [reply src/og/Extent.js]: @yeluoyunfei 
The following annotation breaks jsdoc.
```
@param {[number,number,number]} comment goes here
```
It's better to use this:
```
@param {Array.<number>} comment goes here (exactly 3 entries)
```
So I'm replacing your changes with the above. @Zemledelec

Comment (yeluoyunfei) [reply src/og/Extent.js]: @tbo47  great! didn't aware of this

Comment (yeluoyunfei) [reply src/og/Extent.js]: @tbo47 , besides the array here, there are also some other issue after we enabled type checker:

2021-10-14T06:03:20.9785106Z node_modules/@openglobus/og/types/layer/index.d.ts(6,23): error TS2307: Cannot find module './Layer.js' or its corresponding type declarations.
2021-10-14T06:03:20.9786862Z node_modules/@openglobus/og/types/layer/index.d.ts(7,24): error TS2307: Cannot find module './Vector.js' or its corresponding type declarations.
2021-10-14T06:03:20.9788673Z node_modules/@openglobus/og/types/control/index.d.ts(7,33): error TS2307: Cannot find module './MouseNavigation.js' or its corresponding type declarations.
2021-10-14T06:03:20.9790691Z node_modules/@openglobus/og/types/control/index.d.ts(9,33): error TS2307: Cannot find module './TouchNavigation.js' or its corresponding type declarations.
2021-10-14T06:03:20.9804307Z node_modules/@openglobus/og/types/math/Vec3.d.ts(459,22): error TS2307: Cannot find module './Quat.js' or its corresponding type declarations.
2021-10-14T06:03:20.9812684Z node_modules/@openglobus/og/types/math/index.d.ts(1,23): error TS2307: Cannot find module './Line2.js' or its corresponding type declarations.
2021-10-14T06:03:20.9814231Z node_modules/@openglobus/og/types/math/index.d.ts(2,23): error TS2307: Cannot find module './Line3.js' or its corresponding type declarations.
2021-10-14T06:03:20.9815743Z node_modules/@openglobus/og/types/math/index.d.ts(6,22): error TS2307: Cannot find module './Quat.js' or its corresponding type declarations.
2021-10-14T06:03:20.9817258Z node_modules/@openglobus/og/types/math/index.d.ts(7,21): error TS2307: Cannot find module './Ray.js' or its corresponding type declarations.
2021-10-14T06:03:20.9830924Z node_modules/@openglobus/og/types/index.d.ts(24,26): error TS2307: Cannot find module './renderer/Renderer.js' or its corresponding type declarations.

2021-10-14T06:03:20.9832101Z 3dmap/LocationGlobeMarker.ts(6,20): error TS2749: 'Entity' refers to a value, but is being used as a type here. Did you mean 'typeof Entity'?
2021-10-14T06:03:20.9857631Z 3dmap/Globe.ts(120,97): error TS2749: 'Layer' refers to a value, but is being used as a type here. Did you mean 'typeof Layer'?
2021-10-14T06:03:20.9859441Z 3dmap/Globe.ts(136,89): error TS2749: 'Layer' refers to a value, but is being used as a type here. Did you mean 'typeof Layer'?
2021-10-14T06:03:20.9866436Z 3dmap/Globe.ts(266,28): error TS2749: 'Layer' refers to a value, but is being used as a type here. Did you mean 'typeof Layer'?
2021-10-14T06:03:20.9862921Z 3dmap/Globe.ts(198,26): error TS2749: 'Layer' refers to a value, but is being used as a type here. Did you mean 'typeof Layer'?

2021-10-14T06:03:20.9850987Z 3dmap/Globe.ts(93,22): error TS2339: Property 'events' does not exist on type 'Planet'.
2021-10-14T06:03:20.9852636Z 3dmap/Globe.ts(101,22): error TS2339: Property 'renderer' does not exist on type 'Planet'.
2021-10-14T06:03:20.9854281Z 3dmap/Globe.ts(105,22): error TS2339: Property 'renderer' does not exist on type 'Planet'.
2021-10-14T06:03:20.9855918Z 3dmap/Globe.ts(113,22): error TS2339: Property 'renderer' does not exist on type 'Planet'.
2021-10-14T06:03:20.9861190Z 3dmap/Globe.ts(191,22): error TS2339: Property 'events' does not exist on type 'CanvasTiles'.
2021-10-14T06:03:20.9864717Z 3dmap/Globe.ts(213,31): error TS2339: Property 'renderer' does not exist on type 'Planet'.
2021-10-14T06:03:20.9868155Z 3dmap/Globe.ts(271,31): error TS2339: Property 'renderer' does not exist on type 'Planet'.
2021-10-14T06:03:20.9869758Z 3dmap/Globe.ts(272,31): error TS2339: Property 'renderer' does not exist on type 'Planet'.

2021-10-14T06:03:20.9768222Z node_modules/@openglobus/og/types/Extent.d.ts(43,46): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9773680Z node_modules/@openglobus/og/types/Extent.d.ts(159,48): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9775656Z node_modules/@openglobus/og/types/layer/BaseGeoImage.d.ts(32,32): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9777791Z node_modules/@openglobus/og/types/layer/BaseGeoImage.d.ts(38,32): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9779549Z node_modules/@openglobus/og/types/layer/BaseGeoImage.d.ts(46,38): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9781343Z node_modules/@openglobus/og/types/layer/BaseGeoImage.d.ts(54,38): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9783188Z node_modules/@openglobus/og/types/layer/BaseGeoImage.d.ts(92,45): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9792424Z node_modules/@openglobus/og/types/math/Vec3.d.ts(46,62): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9794253Z node_modules/@openglobus/og/types/math/Vec3.d.ts(55,84): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9797376Z node_modules/@openglobus/og/types/math/Vec3.d.ts(62,25): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9799040Z node_modules/@openglobus/og/types/math/Vec3.d.ts(316,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9800693Z node_modules/@openglobus/og/types/math/Vec3.d.ts(373,21): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9802418Z node_modules/@openglobus/og/types/math/Vec3.d.ts(379,23): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9806764Z node_modules/@openglobus/og/types/math/Vec2.d.ts(215,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9808343Z node_modules/@openglobus/og/types/math/Vec2.d.ts(246,21): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9809789Z node_modules/@openglobus/og/types/math/Vec4.d.ts(85,21): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9811205Z node_modules/@openglobus/og/types/math/Vec4.d.ts(91,23): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9818892Z node_modules/@openglobus/og/types/scene/Planet.d.ts(509,37): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9820261Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(33,43): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9821451Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(48,40): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9822565Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(63,51): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9823858Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(63,275): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9825346Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(84,27): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9826449Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(267,44): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9827525Z node_modules/@openglobus/og/types/entity/Polyline.d.ts(274,36): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9828602Z node_modules/@openglobus/og/types/entity/PointCloud.d.ts(138,36): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).
2021-10-14T06:03:20.9829800Z node_modules/@openglobus/og/types/entity/EntityCollection.d.ts(145,29): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).

Comment (tbo47) [reply src/og/Extent.js]: yes, we need to refactor the code to use es6 `class` syntax.
It's a big effort and I want to write unit tests for that (to make sure I don't create regression while refactoring)
https://github.com/openglobus/openglobus/pull/404

So I go step by step in the right direction.

Comment (yeluoyunfei) [reply src/og/Extent.js]: @tbo47 . Great!  That's the right direction to make unit tests first.

**Review by Zemledelec — APPROVED** [review]

**Pull request merged**